project_earned_value/report/project_report.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2020 Open Source Integrators
+# Copyright (C) 2020 Serpent Consulting Services
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, tools
+
+
+class ReportProjectTaskUser(models.Model):
+    _inherit = "report.project.task.user"
+
+    planned_value = fields.Float('Planned Value', readonly=True)
+    earned_value = fields.Float('Earned Value', readonly=True)
+    actual_cost = fields.Float('Actual Cost', readonly=True)
+    schedule_variance = fields.Float('Schedule Variance', readonly=True)
+    cost_variance = fields.Float('Cost Variance', readonly=True)
+
+    def _select(self):
+        select_str = """
+             SELECT
+                    (select 1 ) AS nbr,
+                    t.id as id,
+                    t.date_start as date_start,
+                    t.date_end as date_end,
+                    t.date_last_stage_update as date_last_stage_update,
+                    t.date_deadline as date_deadline,
+                    t.user_id,
+                    t.project_id,
+                    t.priority,
+                    t.name as name,
+                    t.company_id,
+                    t.partner_id,
+                    t.stage_id as stage_id,
+                    t.kanban_state as state,
+                    t.working_days_close as working_days_close,
+                    t.working_days_open  as working_days_open,
+                    (extract('epoch' from (t.date_deadline-(now() at time zone
+                     'UTC'))))/(3600*24)  as delay_endings_days,
+                    (t.planned_hours * l.price_unit) as planned_value,
+                    (t.planned_hours * s.ev_percent * l.price_unit)
+                        as earned_value,
+                    (t.effective_hours * e.timesheet_cost) as actual_cost,
```

Comment:
This will give wrong result on historical data, once the timesheet cost is updated.
We should get this cost info from the Analytic Lines. Use the `amount` field.

So we need a JOIN with analytic lines, SUM operators on the computations, and the GROUP BY back.